taxi/tests/test_models.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ManufacturerModelTest(TestCase):
+    def setUp(self):
+        self.manufacturer1 = Manufacturer.objects.create(name="Ford",
+                                                         country="USA")
+        self.manufacturer2 = Manufacturer.objects.create(name="BMW",
+                                                         country="Germany")
+
+    def test_manufacturer_str(self):
+        self.assertEqual(str(self.manufacturer1), "Ford USA")
+
+    def test_manufacturer_ordering(self):
+        self.assertEqual(list(Manufacturer.objects.all()),
+                         [self.manufacturer2, self.manufacturer1])

```

Comment:
The `test_manufacturer_ordering` test assumes that manufacturers are ordered by name or some other field. Ensure that the `Manufacturer` model's `Meta` class specifies an ordering attribute that matches this assumption, or adjust the test to reflect the actual ordering defined in the model.